bodhi/server/models.py

```diff
@@ -1999,7 +2008,7 @@ def days_to_stable(self):
         mandatory_days_in_testing or longer."""
         if not self.meets_testing_requirements and self.date_testing:
             return (
-                self.release.mandatory_days_in_testing -
+                self.mandatory_days_in_testing -
                 (datetime.utcnow() - self.date_testing).days)
         return 0
```

Comment:
We need to add a test for this method to prove that the original bug that was filed is now solved. Basically, set up a critical path update and make sure this returns the correct number of days.